app/_data/tables/install_options.yml

```diff
@@ -13,50 +13,60 @@ features:
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/docker.png
       - name: "Debian"
         url: /gateway/VERSION/install/linux/debian/
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/debian-logo.jpg
       - name: "Ubuntu"
         url: /gateway/VERSION/install/linux/ubuntu/
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/ubuntu.png
       - name: "RedHat"
         url: /gateway/VERSION/install/linux/rhel/
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/rhel.jpg
       - name: "Amazon Linux 2"
         url: /gateway/VERSION/install/linux/amazon-linux/
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/amazon-linux.png
   - name: "Kubernetes"
     items:
       - name: "Kubernetes (YAML Manifests)"
         url: /gateway/VERSION/install/kubernetes/kubectl/
         oss: true
         enterprise: true
         support: false
+        icon: /assets/images/icons/documentation/kubernetes-logo.png
       - name: "Kubernetes (Helm)"
         url: /gateway/VERSION/install/kubernetes/helm-quickstart/
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/helm-icon-color.svg
       - name: "OpenShift"
         url: /gateway/VERSION/install/kubernetes/openshift/
         oss: true
         enterprise: true
-        support: false
+        support: true
+        icon: /assets/images/icons/documentation/openshift-logo.png
 
   - name: "Docker"
     items:
       - name: "Debian (bullseye-slim)"
         oss: true
         enterprise: true
         support: true
+        url: /gateway/VERSION/install/docker/
+        icon: /assets/images/icons/documentation/docker.png
       - name: "RHEL (8-ubi)"
```

Comment:
RHEL should have a logo for Docker usage too

Reply:
Do you think we should do Docker icons or distro icons here? I put distro, but I could just do Docker icons for all three.

Reply:
Distro icons look good to me